### DISCUSSION TITLE
Is assignment forking assignment currently not working?

### DISCUSSION THREAD

**Discussion by jintrone** [community/community#116954] [GitHub Education]
### Hi everyone,

I've been using GitHub education all semester; I opted in to forking assignments as soon as the feature was available and have not had any difficulties. However, I recently posted an assignment with a starter repository containing a single .md file; students accepting the assignment have a repo created, but it is empty and does not appear to be a fork of the starter repository.  I've check this twice (creating a new assignment, and accepting it myself) and the problem seems to persist.  I'm wondering if others have encountered this?

**Comment by samus-aran** [comment]
Hi @jintrone 

Are you still experiencing this? We've had another member raise this and stated it was resolved the day after. Is that the same experience as yours?

https://github.com/orgs/community/discussions/116922

**Reply by jintrone** [reply]
Answering your question - yes, a similar experience, although IIRC, an empty README file was created; this may have been because I selected a due date with a cutoff though.  I think I tried twice, once with a README.md file, and upon encountering the problem, tried creating a source repo that had a file with a different name (thinking that the bug could have had something to do with the auto committed due date).  The problem was the same.  When I tried the next day, everything was ok.

**Comment by jintrone** [comment]
Hi, thanks for the reply.  It did appear to resolve itself the next day.

Josh Introne: https://ischool.syr.edu/joshua-introne/
C4 Lab: https://c4-lab.github.io/
Recent Work: Measuring Belief Dynamics on Twitter<http://bit.ly/BeliefDynamics>

On Apr 17, 2024, at 10:44 AM, Ashley Nicolson ***@***.******@***.***>> wrote:



Hi @jintrone<https://github.com/jintrone>

Are you still experiencing this? We've had another member raise this and stated it was resolved the day after. Is that the same experience as yours?

https://github.com/orgs/community/discussions/116922
